### PR TITLE
Update Colorize Theme to use TweenService and task.wait

### PR DESCRIPTION
**Pull request by xvarmkorv2**

### CODE REVIEW

File: MainModule/Client/UI/Colorize.rbxmx [hunk 1]
```diff
@@ -58,12 +58,12 @@
 				waiting = false
 			end
 			
-			yes.MouseButton1Click:Connect(function()
+			yes.MouseButton1Click:connect(function()
```
Comment (P3tray) [inline MainModule/Client/UI/Colorize.rbxmx:61]: Please don't uncaptialize `:Connect()`

File: MainModule/Client/UI/Colorize.rbxmx [hunk 2]
```diff
@@ -58,12 +58,12 @@
...
 				ans = "Yes"
 				gTable:Destroy()
 			end)
 			
-			no.MouseButton1Click:Connect(function()
+			no.MouseButton1Click:connect(function()
```
Comment (P3tray) [inline MainModule/Client/UI/Colorize.rbxmx:66]: Please don't uncaptialize `:Connect()`

File: MainModule/Client/UI/Colorize.rbxmx [hunk 3]
```diff
@@ -99,86 +96,73 @@ return function(gui, guiData, gTable, env)
...
 	}
 	
+	local contents = {}
+	local OrgColors = {}
+	
 	local function getCont(obj)
-		for i,v in pairs(obj:GetChildren()) do 
-			if classes[v.ClassName] then
-				table.insert(contents,v)
+		for i,v in ipairs(obj:GetChildren()) do
```
Comment (P3tray) [inline MainModule/Client/UI/Colorize.rbxmx:103]: We recently removed all pairs() and ipairs() functions from our code.

Pleas change `for i,v in ipairs(obj:GetChildren())` to `for i,v in obj:GetChildren()` and re-test your code.

**Review by P3tray — CHANGES_REQUESTED** [review]
If the code has been tested by yourself and is working, then please make these further changes.

**Review by LawMixer — APPROVED** [review]
I mean, this looks good to me. Just wait for somebody with write access to review and merge it.

**Review by Expertcoderz — APPROVED** [review]

**Pull request merged**